filament/include/filament/RenderableManager.h

```diff
@@ -488,6 +488,9 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
     void setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
             MorphTargetBuffer* morphTargetBuffer, size_t count); //!< \overload
 
+    void setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
+            MorphTargetBuffer* morphTargetBuffer); //!< \overload
+
```

Comment:
This needs a java version.
Maybe it would make more sense to remove the overload with only a count. Also I think this API change should be in a separate PR.

Reply:
Ok I see. I will revert this change now and will prepare a new PR for this.